packages/@react-types/slider/src/index.d.ts

```diff
@@ -2,30 +2,58 @@ import {AriaLabelingProps, AriaValidationProps, FocusableDOMProps, FocusableProp
 import {ReactNode} from 'react';
 
 export interface BaseSliderProps extends RangeInputBase<number>, LabelableProps, AriaLabelingProps {
+  /**
+   * The orientation of the Slider.
+   * @default 'horizontal'
+   */
   orientation?: Orientation,
+  /** Whether the whole Slider is disabled. */
   isDisabled?: boolean,
-  formatOptions?: Intl.NumberFormatOptions
-}
-
-export interface SliderProps extends BaseSliderProps, ValueBase<number[]> {
+  /**
+   * The display format of the value label.
+   */
+  formatOptions?: Intl.NumberFormatOptions,
+  /** Fired when the slider stops moving, due to being let go. */
   onChangeEnd?: (value: number[]) => void
 }
 
+export interface SliderProps extends BaseSliderProps, ValueBase<number[]> {}
+
 export interface SliderThumbProps extends AriaLabelingProps, FocusableDOMProps, FocusableProps, Validation, AriaValidationProps, LabelableProps {
+  /**
+   * The orientation of the Slider.
+   * @default 'horizontal'
+   */
   orientation?: Orientation,
+  /** Whether the Thumb is disabled. */
   isDisabled?: boolean,
+  /** Index of the thumb for accessing purposes. */
   index: number
 }
 
 export interface SpectrumBarSliderBase<T> extends BaseSliderProps, ValueBase<T>, StyleProps {
+  /**
+   * The label's overall position relative to the element it is labeling.
+   * @default 'top'
+   */
   labelPosition?: LabelPosition,
   /** Whether the value's label is displayed. True by default if there's a `label`, false by default if not. */
   showValueLabel?: boolean,
   /** The content to display as the value's label. Overrides default formatted number. */
-  valueLabel?: ReactNode
+  valueLabel?: ReactNode,
+  /**
+   * The slider's minimum value.
+   * @default '0'
+   */
+  minValue?: number,
+  /**
+   * The slider's maximum value.
+   * @default '100'
+   */
+  maxValue?: number
```

Comment:
Should these be moved to SliderProps or replicated there? Just wondering cuz the `minValue`, `maxValue`, and `step` in `useSliderState` docs is missing the default values there 
![image](https://user-images.githubusercontent.com/9661127/102148873-1e1d9f00-3e22-11eb-835e-31f76b4cb450.png)

Reply:
well, so interesting thing about this one, the defaults for all three actually change depending on formatOptions...
I'm not sure how to convey that.

Reply:
wait they do? I can see that the hook can return formatted values but the defaults for `minValue`/`maxValue`/`step` are initialized as 0,100, and 1 respectively right?

Reply:
Typically those would be the values, but if percentage is used they are 0/1/0.01